### ISSUE TITLE
SKTConversationViewController Crash

### ISSUE THREAD

**Issue by hcrub** [smooch/smooch-ios#128]
Receiving a `UITableView` exception from `SKTConversationViewController`.

**Version**
5.3.0 & 5.4.3

**Backtrace**
```objc
#0. Crashed: com.apple.main-thread
0  UIKit                          0x109434d4d -[UITableViewRowData numberOfRowsInSection:] + 44
1  Smooch                         0x108ea0b10 __59-[SKTConversationViewController refreshMessagesWithBounce:]_block_invoke + 154
2  Smooch                         0x108e79164 SKTEnsureMainThread + 91
3  Smooch                         0x108ea0a65 -[SKTConversationViewController refreshMessagesWithBounce:] + 82
4  Smooch                         0x108ea0603 -[SKTConversationViewController newMessagesReceived:] + 217
5  CoreFoundation                 0x1055205ec CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER + 12
6  CoreFoundation                 0x1055204eb _CFXRegistrationPost + 427
7  CoreFoundation                 0x105520252 ___CFXNotificationPost_block_invoke + 50
8  CoreFoundation                 0x1054e3282 -[_CFXNotificationRegistrar find:object:observer:enumerator:] + 2018
9  CoreFoundation                 0x1054e231b _CFXNotificationPost + 667
10 Foundation                     0x10670581b -[NSNotificationCenter postNotificationName:object:userInfo:] + 66
11 Smooch                         0x108e9ba19 __42-[SKTConversation notifyMessagesReceived:]_block_invoke + 307
12 libdispatch.dylib              0x10c2fb12e _dispatch_client_callout + 8
13 libdispatch.dylib              0x10c2e3773 _dispatch_barrier_sync_f_slow_invoke + 315
14 libdispatch.dylib              0x10c2fb12e _dispatch_client_callout + 8
15 libdispatch.dylib              0x10c2e24fb _dispatch_main_queue_callback_4CF + 1054
16 CoreFoundation                 0x105546e49 CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE + 9
17 CoreFoundation                 0x10550c37d __CFRunLoopRun + 2205
18 CoreFoundation                 0x10550b884 CFRunLoopRunSpecific + 420
19 GraphicsServices               0x10d104a6f GSEventRunModal + 161
20 UIKit                          0x10907fc68 UIApplicationMain + 159
21 App                           0x102ab8f20 main (main.m:25)
22 libdyld.dylib                  0x10c34468d start + 1
```

**Setup**
We have a simple UITabBarController whereas we're using a SKTConversationViewController in one of our tabs.

```objc
UIViewController *smoochViewController = [Smooch newConversationViewController];
UINavigationController *smoochNavigationController = [[UINavigationController alloc] initWithRootViewController:smoochViewController];
self.tabBarController.viewControllers = @[nav1, nav2, smoochNavigationController, nav4];
```

**Reproducing**
Quickly switch between the Smooch View Controller tab and another View Controller tab until you receive a crash.

**Comment by wmora** [comment]
Thanks for reporting. We'll look into it

**Comment by wmora** [comment]
This issue is fixed in [v5.4.4](https://github.com/smooch/smooch-ios/releases/tag/5.4.4)
